pkg/base1/cockpit.js

```diff
@@ -2808,7 +2826,6 @@ function full_scope(cockpit, $, po) {
                     var message_len = message.length;
                     if (message_len === undefined) {
                         meta = message;
-                        last = null;
```

Comment:
I thought all bets were off when a meta message came through. The data before/after can be discontiguous. Could you explain why this line is removed?